### PR TITLE
Adds check to forbid auto-names for migrations

### PR DESCRIPTION
**Pull request by sobolevn**
Features:
- New docs (with credits and links back to the original author)
- New installable app
- New tests: unit and integration ones

Closes #34

### CODE REVIEW

**Comment by sobolevn** [comment]
@adamchainz I would be happy if you have the time to review this PR. It is completely based on your blog post.

Thanks a lot for writing it! I love your blog ❤️

**Comment by adamchainz** [comment]
👍 👍 👍

**Comment by adamchainz** [comment]
Imo it couldˆ do with the setting for an exclusion list of allowed badly named migrations, otherwise it's impossible to add into new projects.

Ping me when this is released and I'll add to the post.

P.S. delete-merged-branch bot can be replaced with github repo setting "Automatically delete head branches"

**Comment by sobolevn** [comment]
@adamchainz thanks! There's already an option for that: https://github.com/wemake-services/django-test-migrations/blob/master/django_test_migrations/autonames.py#L30

But, I will also add it to the docs. 👍

**Comment by sobolevn** [comment]
@adamchainz release is published: https://github.com/wemake-services/django-test-migrations/releases/tag/0.2.0

**Comment by adamchainz** [comment]
Added link to post

File: tests/test_autonames/conftest.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import pytest
+
+
+@pytest.fixture()
+def ignore_migration(settings):
+    """We patch settings to ignore this one bad migration."""
+    settings.DTM_IGNORED_MIGRATIONS = {
+        ('main_app', '0004_auto_20191119_2125'),
+    }
+    return settings
```
Comment (blueyed) [inline tests/test_autonames/conftest.py]: I think having this directly in the test (instead of a fixture in another file) would be clearer.

Comment (sobolevn) [reply tests/test_autonames/conftest.py]: Fair enough! Scheduled: https://github.com/wemake-services/django-test-migrations/issues/36

**Pull request merged**